src/main/java/nextstep/subway/applicaion/LineService.java

```diff
@@ -55,17 +62,10 @@ public LineResponse getLine(Long id) {
         );
     }
 
-    public LineResponse updateLine(Long id, LineRequest lineRequest) {
+    public void updateLine(Long id, LineRequest lineRequest) {
         Line line = lineRepository.getById(id);
-        line.modify(lineRequest.getName(), lineRequest.getColor());
-        lineRepository.save(line);
-        return new LineResponse(
-                line.getId(),
-                line.getName(),
-                line.getColor(),
-                line.getCreatedDate(),
-                line.getModifiedDate()
-        );
+        line.setName(lineRequest.getName());
+        line.setColor(lineRequest.getColor());
```

Comment:
setter를 이용한 access method 사용은 지양해야 합니다 ㅠㅠ 
public한 setter를 선언하는순간 외부 의도치 않은 영역에서 객체의 위변조가 일어날 가능성이 생기기에 객체의 불변성을 최대한 확보하려는 것이고, 요구사항에 의해서 값이 수정되야하는경우 의미있는 메서드(ex: update, change, ...etc)를 만들어 내부에서 개발자에 의해 철저히 통제되는 변경 로직을 작성하는 것입니다. 

```suggestion
        line.update(lineRequest.getName(), lineRequest.getColor()); 이게 더 맞겠죠. 
```
더 복잡하게 생각하면 믹스인 인터페이스 혹은 lambda를 이용할수도 있지만, 이 경우에는 간단하게 위처럼 작성할 수 있을 것 같아요.

Reply:
https://github.com/next-step/atdd-subway-map/pull/192#discussion_r790208455

아앗,,,나누는게 좋은것 같다는 리뷰인줄알았더니 제가 잘못 봤네요 ㅠ,ㅠ!! 
리뷰 감사합니다!